package.json

```
@@ -3,12 +3,16 @@
   "version": "2.0.0",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js"
+    "start": "node app.js",
+    "dev": "nodemon app.js"
```

```suggestion
    "dev": "nodemon -e hbs,js,css app.js"
```

Para asegurarnos que pilla los cambios pero vamos, minucias